mkdocs.yml

```diff
@@ -33,6 +33,7 @@ pages:
     - Capabilities: 'datastorage/capabilities.md'
     - World Saved Data: 'datastorage/worldsaveddata.md'
     - Extended Entity Properties: 'datastorage/extendedentityproperties.md'
+    - Config: 'config/@config.md'
```

Comment:
If possible the name should be changed to `@Config` because ideally `Configuration` would also be documented somewhere and it could be confusing.

Reply:
If it breaks it's not my fault 😛